docs/localInstallation.md

```diff
@@ -495,9 +495,24 @@ If problems can't be resolved like that please go to the [current error](#errors
 
 This section isn't necessary doing with netbeans. If netbeans running slowly on your pc you can use a classical text editor (nano, vim, gedit...) to edit configuration files.
```

Comment:
You don't need to edit your configuration files specifically with netbeans, you can use a classical text editor (nano, vim, gedit...) to edit them.